ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!---
+Hi! This template is here to help us collect optimal issue reports. Feel free to adapt it to your situation.
+First, check whether the issue has already been reported. If so, please add this report to that issue thread. If not, proceed to file this new issue.
+Next, please provide a one to four sentence summary of the issue. If you do your very best to be clear, concise and accurate here, other Rustaceans will be eager to assist. Also try to mention which GitHub labels this issue likely needs and/or what parts of Rust it relates to.
```

Comment:
I’m pretty sure people who are triaging the issues are much more suited for the task of triaging the issue compared to the average compiler user. 

Most of them see compiler as a black box which gets fed source files in one end, and binaries, after some time, come out from another.